### ISSUE TITLE
Make sure Ace doesn't report DPUB ARIA roles as violations

### ISSUE THREAD

**Issue by rdeltour** [daisy/ace#65]
aXe currently reports DPUB ARIA roles as critical violations (in its `aria-roles` rule). Ace shouldn't report these as violations.

Possible solutions:
- override the `aria-roles` rule with a custom implementation
- submit a PR to aXe to support DPUB ARIA roles

**Comment by JayPanoz** [comment]
Yeah, that would be very useful. 

I’ve just tested a complex epub with lots of footnotes, bib ref, etc., and literally wailed in fear and begged to be forgiven when checking the results. 

Then I noticed that, 2742 of 2743 violations were about DPUB aria-roles but it was kinda tricky to check the results (I had to parse that first to eliminate those violations).

**Comment by rdeltour** [comment]
> literally wailed in fear and begged to be forgiven when checking the results.

oh, sorry about the cold sweat 😖!

**Comment by JayPanoz** [comment]
Nah don’t worry. Of all the files I could test, I picked the edge case. ;-)

**Comment by rdeltour** [comment]
Reported in dequelabs/axe-core#580

**Comment by LaZay** [comment]
I have just tested an ePub with ACE Beta version myself.
And I confirm. All DPUB ARIA roles are reported as critical errors, Which is quite ironic as I added these roles on purpose in order to systematically provide ARIA equivalents to epub:type attribute.
https://www.w3.org/TR/dpub-aria-1.0/#roles
